Resources/views/Collector/db.html.twig

```diff
@@ -267,6 +272,36 @@
                                 {% if query.explainable %}
                                     <div id="explain-{{ i }}-{{ loop.parent.loop.index }}" class="sql-explain"></div>
                                 {% endif %}
+
+                                {% if query.backtrace is defined %}
+                                    <div id="backtrace-query-{{ i }}-{{ loop.parent.loop.index }}" class="hidden">
+                                        <table>
+                                            <thead>
+                                                <tr >
+                                                    <th scope="col">#</th>
+                                                    <th scope="col">File/Call</th>
+                                                </tr>
+                                            </thead>
+                                            <tbody>
+                                                {% for trace in query.backtrace %}
+                                                    <tr>
+                                                        <td>{{ loop.index }}</td>
+                                                        <td>
+                                                            <span class="text-small">
+                                                                {% set line_number = trace.line|default(1) %}
+                                                                {% set file_path = trace.file|format_file(line_number)|striptags|replace({ (' at line ' ~ line_number): '' }) %}
```

Comment:
why using `|format_file(line_number)` if you remove both the tags (making the file clickable) and the line number ?

Reply:
it's for making relative paths

Reply:
but why not keeping the full feature of `|format_file` which makes them clickable ?